client/src/components/RelatedObjectNotes.js

```diff
@@ -276,11 +276,24 @@ const RelatedObjectNotes = ({
                           .filter(
                             field => !EXCLUDED_ASSESSMENT_FIELDS.includes(field)
                           )
-                          .map(field => (
-                            <p key={field}>
-                              <i>{field}</i>: <b>{jsonFields[field]}</b>
-                            </p>
-                          ))}
+                          .map(field => {
+                            const isRichTextField = utils.isValidHtmlWithTags(
+                              jsonFields[field]
+                            )
+                            const fieldValue = isRichTextField
+                              ? parseHtmlWithLinkTo(jsonFields[field])
+                              : jsonFields[field]
```

Comment:
What if the field contains an array (where each item in the array could have a rich-text element)?